ports/apocrunner/port.json

```
@@ -1,27 +1,34 @@
 {
-    "version": 2,
-    "name": "apocrunner.zip",
-    "items": [
-        "Apoc Runner.sh",
-        "apocrunner"
+  "version": 3,
+  "name": "apocrunner.zip",
+  "items": [
+    "Apoc Runner.sh",
+    "apocrunner"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Apoc Runner",
+    "porter": [
+      "mattyj513",
+      "t0b10"
     ],
-    "items_opt": [],
-    "attr": {
-        "title": "Apoc Runner",
-        "porter": [
-            "mattyj513"
-        ],
-        "desc": "As the world burns, as the streets crumble, you cling to one thing: Survival. An unstoppable alien blight has taken hold of the earth, and there's no fighting it. Escape is the only option. Run, jump, and slide your way through urban destruction in this intense runner!",
-        "inst": "Download game from https://omnitron-studios.itch.io/apoc-runner, place data.win in /ports/apocrunner/gamedata",
-        "genres": [
-            "arcade"
-        ],
-        "image": null,
-        "rtr": false,
-        "runtime": null,
-        "reqs": [],
-        "arch": [
-            "armhf"
-        ]
-    }
-}
\ No newline at end of file
+    "desc": "As the world burns, as the streets crumble, you cling to one thing: Survival. An unstoppable alien blight has taken hold of the earth, and there's no fighting it. Escape is the only option. Run, jump, and slide your way through urban destruction in this intense runner!",
+    "desc_md": "As the world burns, as the streets crumble, you cling to one thing: **Survival.** An unstoppable alien blight has taken hold of the earth, and there's no fighting it. **Escape** is the **only option**. Run, jump, and slide your way through urban destruction in this intense runner!",
+    "inst": "Thanks to Omnitron Studios, there is no need to download the game from itch.io to obtain the game files. This port is 'Ready to Run' as soon as it's been installed on your device.",
+    "inst_md": "Thanks to [**Omnitron Studios**](https://omnitron-studios.itch.io), there is no need to download the game from [itch.io](https://omnitron-studios.itch.io/apoc-runner) to obtain the game files. This port is now *Ready to Run* as soon as it's been installed on your device.",
+    "genres": [
+      "action",
+      "arcade",
+      "platformer"
+    ],
+    "image": "screenshot.png",
```

This field is set when PortMaster build the release.
```suggestion
    "image": null,
```